.bingo/prometheus.mod

```diff
@@ -19,4 +19,50 @@ replace (
 	k8s.io/klog => github.com/simonpasquier/klog-gokit v0.1.0
 )
 
-require github.com/prometheus/prometheus v2.4.3+incompatible // cmd/prometheus
+require (
```

Comment:
Was this manually changed?

Reply:
Let's remove this file change otherwise - this is managed by `bingo` tool

Reply:
Done... I think this file was updated by one of the various make commands, it was manually staged in the PR.